src/backend/api/reservations.js

```diff
@@ -1,16 +1,9 @@
+const { request, response } = require("express");
 const express = require("express");
 const router = express.Router();
 const knex = require("../database");
 
-router.get("/", async (request, response) => {
-  try {
-    // knex syntax for selecting things. Look up the documentation for knex for further info
-    const titles = await knex("reservations").select("title");
-    response.json(titles);
-  } catch (error) {
-    throw error;
-  }
-});
+
 //Returns all reservations
 router.get("/", async (request, response) => {
   try {
```

Comment:
i don't know why i can not leave comments on lines further than 9 but no problem 
covert it to Number because it is by default string 
  const id = parseInt(request.params.id);

Reply:
thanks I will fix that, but can you help me with one thing , I have problem with @media query, do you know how I can fix it ? if you check my app in your phone you will see somehow menu bar is not working right :(!!!!